frontends/web/src/routes/account/info/buyReceiveCTA.tsx

```diff
@@ -45,7 +45,7 @@ export const BuyReceiveCTA = ({ code, unit, balanceList, exchangeBuySupported =
         route('accounts/select-receive');
         return;
       }
-      route(`/account/${balanceList[0]}/receive`);
+      route(`/account/${code}/receive`);
```

Comment:
We could have same issue in other places.
I think we never want `undefined` in template literals could that be an eslint or ts rule somehow?